server/public/dashboard/dashboard.html

```diff
@@ -32,30 +32,29 @@
 <!-- List of Emergencies (Right Side) -->
 <md-content class="md-padding" id="alert_container">
 
-  <!-- Toggle Emergencies (Active or Inactive) -->
-  <div class="emergencyToggle">
-    <md-switch ng-init="dashboard.showInactive = false" ng-model="dashboard.showInactive" class="md-primary" md-no-ink>Show Inactive Emergencies</md-switch>
-  </div>
-
-  <!-- Logout Button -->
-  <div align="right">
-    <md-button class="md-raised" ng-click="dashboard.logout()">Logout</md-button>
-  </div>
-
-  <!-- Heatmap Toggle -->
-  <div align="right">
-    <md-button class="md-raised" ng-click="dashboard.toggleHeatMap()">Toggle Heatmap</md-button>
-  </div>
-
-  <!-- Fake Emergency Button (For Debugging)-->
-  <!--
-  <div alight="right">
-    <md-button class="md-raised" ng-click="dashboard.fakeEmergency()">Fake Emergency</md-button>
-  </div>
-  -->
+  <!-- People Search -->
+  <md-input-container md-no-autogrow="true" class="search">
+    <input type="text" ng-model="searchText" placeholder="Person or Emergency ID">
+  </md-input-container>
+
+    <!-- Logout Button -->
+    <md-button class="md-raised logout_btn" ng-click="dashboard.logout()">Logout</md-button>
+    <div layout="row">
+      <div flex="50" class="checkboxes">
+   
+        <!-- Toggle Emergencies (Active) -->
+        <md-checkbox class="md-primary" ng-model="dashboard.showActive" aria-label="Active">Active</md-checkbox>
+      
+        <!-- Toggle Emergencies (Inactive) -->
+        <md-checkbox class="md-primary" ng-model="dashboard.showInactive" aria-label="Inactive">Inactive</md-checkbox>
+      </div>
+
+      <!-- Heatmap Toggle -->
+      <md-switch ng-init="dashboard.showHeatMap = false" ng-model="dashboard.showHeatMap" class="md-primary heatmap_toggle" md-no-ink>Show Heat Map</md-switch>
+    </div>
 
   <!-- Emergencies (Active) -->
-  <div layout="column" layout-gt-xs="row" ng-if="!dashboard.showInactive && emergency.active" ng-repeat="emergency in dashboard.emergencies | orderBy: '-started'">
+  <div layout="column" layout-gt-xs="row" ng-if="dashboard.showActive && emergency.active" ng-repeat="emergency in dashboard.emergencies | filter: searchText | orderBy: '-started'">
```

Comment:
This looks good, but in Angular you can specify which key you want to search. Right now your filtering by all values in the emergencies, which could be slower.